challenge_final/launch/amigo_dependencies.launch

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```

Comment:
Not used in final, plus not a real feature at the moment, so could be removed in my opinion